lib/plugins/aws/package/compile/functions.js

```diff
@@ -114,6 +129,16 @@ class AwsCompileFunctions {
     });
   }
 
+  async addFileToHash(filePath, hash) {
+    const lambdaHashingVersion = this.serverless.service.provider.lambdaHashingVersion;
+    if (lambdaHashingVersion === '20201221') {
```

Comment:
I think it's fine to check `if (lambdaHashingVersion)`, also I'd expect in eventual future improvements, we may to do comparisions as `if (lambdaHashingVersion >= 20201221)` (so given logic applies since given point in time, and not just for given specific improvement)

Reply:
Here I also wasn't sure about this, I was even pondering with having a helper that returns correct handler based on `lambdaHashingVersion` but it seemed like overkill. I like the suggestion, I also think we won't be doing these changes very often.